### PR TITLE
docs(mantis-guides): add team staking & restaking guides

### PR DESCRIPTION
**Pull request by JafarAz**
Required for merge:
- [ ] `pr-workflow-check / draft-release-check` is ✅ success
- Other rules GitHub shows you, or can be read in [configuration](github.com/ComposableFi/env/terraform/github.com/branches.tf) 

Makes review faster:
- [x] PR title is my best effort to provide summary of changes and has clear text to be part of release notes 
- [x] I marked PR by `misc` label if it should not be in release notes
- [ ] Linked Zenhub/Github/Slack/etc reference if one exists
- [ ] I was clear on what type of deployment required to release my changes (node, runtime, contract, indexer, on chain operation, frontend, infrastructure) if any in PR title or description
- [ ] Added reviewer into `Reviewers`
- [ ] I tagged(`@`) or used other form of notification of one person who I think can handle best review of this PR
- [ ] I have proved that PR has no general regressions of relevant features and processes required to release into production
- [ ] Any dependency updates made, was done according guides from relevant dependency
- Clicking all checkboxes 
- Adding detailed description of changes when it feels appropriate (for example when PR is big)

### CODE REVIEW

File: docs/docs/technology/solana-restaking/mantis-games.md [hunk 1]
```diff
@@ -24,5 +24,14 @@ There will be 17,500,000 PICA allocated to reward vault participants. Once the S
 
 The team that wins the competition, will receive 80% of transfer revenue for 30 days.
```
Comment (blasrodri) [inline docs/docs/technology/solana-restaking/mantis-games.md:26]: with the launch of the Solana<>Cosmos IBC bridge

File: docs/docs/technology/solana-restaking/mantis-games.md [hunk 2]
```diff
@@ -24,5 +24,14 @@ There will be 17,500,000 PICA allocated to reward vault participants. Once the S
...
 
+The Restaking Vaults feature a reward pool of 17.5 million PICA tokens. Notably, 20% of this pool, equivalent to 3.5 million PICA, will be distributed during the Token Generation Event (TGE) of PICA on Solana coinciding with the SOL IBC launch. The remaining 80%, totalling 14 million PICA, will be vested evenly over a 3-month period following the conclusion of the team competition. Monthly distributions will average around 4.66 million PICA.
+
+**It’s essential that users are aware of the implications of entering the vaults and ramifications on their rewards, should they choose to unstake before rewards have fully vested.** During the MANTIS Games Competition (restaking vaults), user assets will be locked within the contracts until the launch of SOL IBC.
+
+Once SOL IBC is launched, users will be able to unstake their assets, but their rewards will be penalized under the following conditions:
```
Comment (blasrodri) [inline docs/docs/technology/solana-restaking/mantis-games.md:31]: don't use SOL for Solana

File: docs/docs/user-guides/restaking-sol.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# How to Restake your tokens 
+
+The [MANTIS app](https://games.mantis.app/) enables you to restake your assets effortlessly using intents. You will need a Phantom wallet to interact with the application. Just prompt the token and amount you want to stake or follow the suggestions.
```
Comment (blasrodri) [inline docs/docs/user-guides/restaking-sol.md:3]: enables the user? do we speak in second person?

Comment (JafarAz) [reply docs/docs/user-guides/restaking-sol.md:3]: for the guides yeah

File: docs/docs/user-guides/restaking-sol.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+The [MANTIS app](https://games.mantis.app/) enables you to restake your assets effortlessly using intents. You will need a Phantom wallet to interact with the application. Just prompt the token and amount you want to stake or follow the suggestions. 
+
+After connecting your wallet, you will see the following screen. Currently, you can either restake bSOL, mSOL, jitoSOL or SOL (staking SOL occurs via converting SOL into wSOL and then transferring it to the vault). 
+
+![stake-dropdown](../user-guides/images-restaking-guide/restaking-1.png)
+
+Once your intent is expressed as displayed below, you’ll need to approve the transaction with your wallet. 
+
+:::info
+The fee to mint an NFT is 0.03 SOL, this amount will be returned to your account once the NFT is burned. You can read more about this process [here](../technology/solana-restaking/vaults.md).
```
Comment (blasrodri) [inline docs/docs/user-guides/restaking-sol.md:12]: shall we explain what these NFTs are for?

Comment (blasrodri) [reply docs/docs/user-guides/restaking-sol.md:12]: given that there are two NFTs, it can be confusing

Comment (JafarAz) [reply docs/docs/user-guides/restaking-sol.md:12]: it's explained in the linked section, not in the user guide

File: docs/docs/technology/solana-restaking/mantis-games.md (hunk 2 above)
Comment (JafarAz) [reply docs/docs/technology/solana-restaking/mantis-games.md]: edited

**Review by blasrodri — APPROVED** [review]

**Pull request merged**